doc/dev.md

```diff
@@ -117,4 +117,124 @@ java -cp bench.jar lrsql.bench [arguments]
 
 Sample insert and query inputs can be found in the distribution at `bench/`
 
+### Reaction JSON
```

Comment:
I'm honestly not sure if this the dev documentation page is best place to put the reactions spec, since it makes it pretty long. Instead would it be a better idea to make a separate page for the spec?

Reply:
I like separate page better I think

Reply:
Also, it is technically not _just_ a private API, because we describe all the other "private" endpoints pretty well. So maybe the reactions spec goes in a subfolder of the main docs?

Reply:
https://github.com/yetanalytics/lrsql/pull/428/commits/f27d6bcedc40de373c6e011672a31c9cba5f4c3c